peripherals/rcc/rcc_f0.yaml

```diff
@@ -350,3 +385,57 @@ RCC:
       Reset: [1, "Reset I/O port B"]
     IOPARST:
       Reset: [1, "Reset I/O port A"]
+  CFGR2:
+    PREDIV:
+      NotDivided: [0, "PREDIV input clock not divided"]
+      Div_2: [1, "PREDIV input clock divided by 2"]
+      Div_3: [2, "PREDIV input clock divided by 3"]
+      Div_4: [3, "PREDIV input clock divided by 4"]
+      Div_5: [4, "PREDIV input clock divided by 5"]
+      Div_6: [5, "PREDIV input clock divided by 6"]
+      Div_7: [6, "PREDIV input clock divided by 7"]
+      Div_8: [7, "PREDIV input clock divided by 8"]
+      Div_9: [8, "PREDIV input clock divided by 9"]
+      Div_10: [9, "PREDIV input clock divided by 10"]
+      Div_11: [10, "PREDIV input clock divided by 11"]
+      Div_12: [11, "PREDIV input clock divided by 12"]
+      Div_13: [12, "PREDIV input clock divided by 13"]
+      Div_14: [13, "PREDIV input clock divided by 14"]
+      Div_15: [14, "PREDIV input clock divided by 15"]
+      Div_16: [15, "PREDIV input clock divided by 16"]
+  CFGR3:
+    "USART*SW":
+      PCLK: [0, "PCLK selected as USART clock source"]
+      SYSCLK: [1, "SYSCLK selected as USART clock source"]
+      LSE: [2, "LSE selected as USART clock source"]
+      HSI: [3, "HSI selected as USART clock source"]
+    USBSW:
+      HSI48: [0, "HSI48 selected as USB clock source"]
```

Comment:
Annoyingly on the F0x0 a `0` here is "USB clock disabled" rather than "HSI48 selected as USB clock source" according to RM0360. We could take this field out of this file and split it into two files, one for the F0x0 and one for the rest of the family?

Reply:
BTW, I have a patch set mostly ready which among other things is adding documentation for RCC fields of F0x1, F0x2 and F0x8 which are not in F0x0.
I was just waiting for this PR to be merged to rebase my patch set and send a new PR.
Having different documentation for USBSW/HSI48 would be trivial after my patch set.
So if you want, you could just merge this PR as is and I will add a fix for this USBSW in my patch set.

Reply:
Perfect, let's do that. Thanks.